sdk/monitor/Azure.Monitor.Query/README.md

```diff
@@ -223,6 +223,33 @@ foreach (var resourceGroup in response.Value)
 }
 ```
 
+### Querying additional workspaces
```

Comment:
```suggestion
### Query additional workspaces
```

Also, please add this section's link to the list beneath the "Examples" heading.